android/build.gradle

```
@@ -30,6 +30,10 @@ if( newDbSize != null && newDbSize.isLong()) {
     dbSizeInMB = newDbSize.toLong()
 }
 
+def useAsyncTaskThreadPoolExecutor = rootProject.hasProperty('AsyncStorage_use_AsyncTaskThreadPoolExecutor') ? rootProject.properties['AsyncStorage_use_AsyncTaskThreadPoolExecutor'] : true
```

Nit: I'm not really fond of explicitly returning `true`/`false` in if-else statements.
```suggestion
def useAsyncTaskThreadPoolExecutor = !rootProject.hasProperty('AsyncStorage_use_AsyncTaskThreadPoolExecutor') || rootProject.properties['AsyncStorage_use_AsyncTaskThreadPoolExecutor']
```